Page Layouts - Final review polish

halocline opened the pull request:
<!--- Provide a general summary of the PR in the Title above -->


<!--- Insert the PR's # for the deploy preview's URL -->
[Deploy Preview](https://deploy-preview-INSERT_PR_#_HERE--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?

#### Where should the reviewer start?

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### How should this be manually tested?

#### Any background context you want to provide?

#### What are the relevant issues?

#### Screenshots (if appropriate)

#### Should this PR be mentioned in Design System updates?

#### Is this change backwards compatible or is it a breaking change?

taysea reviewed (commented):
Overall this looks good and nice adjustments to the relatedContent. Just a couple things I noticed as I was scaling the screen down:

1. The right column starts to overlap the left (this is very briefly before a breakpoint shift):

<img width="605" alt="Screen Shot 2022-02-10 at 9 43 01 PM" src="https://user-images.githubusercontent.com/12522275/153484610-299625c9-02be-465a-8bd9-f2aafdf957c3.png">

2. On (I think) "medium" screen, the yellow background isn't filling the full vertical height of its card:

<img width="225" alt="Screen Shot 2022-02-10 at 9 43 14 PM" src="https://user-images.githubusercontent.com/12522275/153484710-92d25716-075a-47bd-a5b4-cfea995a3a41.png">

3. The elevation of the bottom cards is still getting cut off -- I believe you had put up a fix for this in a previous PR but maybe it's not being reflected here?

<img width="647" alt="Screen Shot 2022-02-10 at 9 43 35 PM" src="https://user-images.githubusercontent.com/12522275/153484975-4828214c-00e7-41a4-8bd5-b3fb7f931099.png">

halocline commented:
> Overall this looks good and nice adjustments to the relatedContent. Just a couple things I noticed as I was scaling the screen down:
> 
> 1. The right column starts to overlap the left (this is very briefly before a breakpoint shift):
> 

I put up a modification here. Let me know if you like it better.

This one is kind of a pros/cons scenario: 
- **before** had overlap for ~1% of the screen widths (14px / 1536px), but the "Server Health" card grid looks 100%.
- **after** 0 overlap, but "Server Health" & "Needs Attention" present awkwardly for ~10% of screen widths (158px / 1536px)
 
> 2. On (I think) "medium" screen, the yellow background isn't filling the full vertical height of its card:
> 

Fixed

> 3. The elevation of the bottom cards is still getting cut off -- I believe you had put up a fix for this in a previous PR but maybe it's not being reflected here?
> 
> 

Fixed

taysea commented:
> I put up a modification here. Let me know if you like it better.
> 
> This one is kind of a pros/cons scenario:
> 
> * **before** had overlap for ~1% of the screen widths (14px / 1536px), but the "Server Health" card grid looks 100%.
> * **after** 0 overlap, but "Server Health" & "Needs Attention" present awkwardly for ~10% of screen widths (158px / 1536px)
> 

Thanks for listing pros/cons. Think the percentages in "before" case are more favorable, so I'll let you decide if you'd prefer to keep it as it was before.
